src/Nox.Types.Abstractions/Types/CultureCode.cs

```diff
@@ -1,7 +1,22 @@
-﻿namespace Nox.Types.Abstractions
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using Nox.Yaml.Attributes;
+
+namespace Nox.Types.Abstractions
 {
-    public class CultureCode
+    public static class CultureCode
     {
         public const string RegularExpression = @"^[a-z]{2}(?:-[A-Z]{2})?(?:-[A-Z][a-z]{3})?$";
+        
+        public static readonly ImmutableSortedDictionary<string,Culture> CultureCodeDisplayNames = Enum.GetValues(typeof(Culture))
+            .Cast<Culture>()
+            .ToImmutableSortedDictionary(x =>
+                {
+                    var field = x.GetType().GetField(x.ToString())!;
```

Comment:
Use the extension method...

```suggestion
                    x.ToDisplayName();
```

Reply:
Desired changens have been applied